src/medUtilities/medUtilities.h

```diff
@@ -59,13 +59,23 @@ class MEDUTILITIES_EXPORT medUtilities
      * @brief Retrieves an array from input data. This functions first looks in
      * point data, then cell data and finally in field data.
      *
-     * @param[in] data input data, must be a mesh or map 
+     * @param[in] data input data, must be a mesh or map
      * @param[in] arrayName array to retrieve
      * @return specified array if it exits, nullptr otherwise
      */
     static vtkDataArray* getArray(dtkSmartPointer<medAbstractData> data,
                                   QString arrayName);
 
+    /**
+     * @brief Return array index and data type
+     * @param[in] data input data, must be a mesh or map
+     * @param[in] arrayName array name
+     * @return data index and data type (0 point data, 1 cell data, 2
+     *         field data) as list
+     */
+    static QList<long> getArrayIndex(dtkSmartPointer<medAbstractData> data,
```

Comment:
Instead of returning a list containing both the index and an arbitrary number giving the array type, it would be better to return only the index and use an output argument (reference or pointer) to **optionally** provide the array type in the form of an enum.
Also:
- Why use long when VTK uses int ?
- I just noticed that all the recently added functions in this class take smart pointers but they don't need to since they don't store the data. Taking a smart pointer instead of a raw pointer makes it complicated for code not using smart pointers to call this.

Reply:
(Actually it's better for the output argument to be a pointer defaulting to null instead of a reference because as I said it should be an optional argument. Some code might already know the array type and it would be useless to force the caller to create an int just to be able to call the function)